mustache.go

```diff
@@ -121,6 +121,28 @@ func (n *sectionNode) render(t *Template, w *writer, c ...interface{}) error {
 	return fmt.Errorf("failed to lookup %s", n.name)
 }
 
+// The testNode type is a complex node which recursively renders its child
+// elements while passing along its context along with the global context.
+type testNode struct {
+	testIdent string
+	testVal   string
+	elems     []node
+}
+
+func (n *testNode) render(t *Template, w *writer, c ...interface{}) error {
+	w.tag()
+	defer w.tag()
+	v, _ := lookup(n.testIdent, c...)
```

Comment:
Could the lookup technically fail and v == nil?

Reply:
Yes, but the way that print handles that it doesn't matter. 

It would result in the string value "null" unless there was a stringer on the struct doing something different.

Reply:
I guess I can make this do a nil check and basically always not match if it was nil.

Reply:
Yeah a nil check would be better.